packages/melody-plugin-load-functions/__tests__/__snapshots__/LoadFunctionSpec.js.snap

```diff
@@ -0,0 +1,32 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`load should compile to import() 1`] = `
+"export const _template = {};
+
+_template.render = function (_context) {
+  import(\\"fancy-component\\");
```

Comment:
Is this supposed to render `fancy-component` ? or just to load the component? What is the use case ?

Reply:
It supposed to only load the component without rendering it. That'll help with conditionally loading custom-elements in the view-layer.